src/suricata.c

```diff
@@ -1458,21 +1461,21 @@ static TmEcode ParseCommandLine(int argc, char** argv, SCInstance *suri)
 #endif /* HAVE_PFRING */
             }
             else if (strcmp((long_opts[option_index]).name , "capture-plugin") == 0){
-                suri->run_mode = RUNMODE_PLUGIN;
-                suri->capture_plugin_name = optarg;
+                suricata.run_mode = RUNMODE_PLUGIN;
+                suricata.capture_plugin_name = optarg;
```

Comment:
Hmm... Might have been simpler to have added at the top of the function:
```
SCInstance *suri = &suricata;
```

Reply:
yeah I think I prefer that for smaller patches initially